cloud/misc/ovirt.py

```diff
@@ -347,6 +413,14 @@ def main():
             disk_int = dict(default='virtio', choices=['virtio', 'ide']),
             instance_os = dict(aliases=['vmos']),
             instance_cores = dict(default=1, aliases=['vmcores']),
+            instance_hostname = dict(aliases=['hostname']),
+            instance_ip = dict(aliases=['ip']),
+            instance_netmask = dict(aliases=['netmask']),
+            instance_gateway = dict(aliases=['gateway']),
+            instance_domain = dict(aliases=['domain']),
+            instance_dns = dict(aliases=['dns']),
+            instance_rootpw = dict(aliases=['rootpw']),
```

Comment:
could you add `no_log=True` so the pw won't be exposed in logs or error messages?